try:
    z=int('iii')
except ValueError as val:
    print(val)
